src/client/components/Title/TitleDesc.styles.css

```diff
@@ -0,0 +1,20 @@
+.title{
+    font-family: 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-size: 30px;
+    font-weight: bold;
+    line-height: 47px;
+    color: rgb(14, 13, 13);
+    margin-bottom: 0rem;
+    }
+    ​
+  .description{ 
+      font-family: 'Times New Roman', Times, serif;
+      padding-top: 1em;
+      font-style: normal;
+      font-size: 18px;
+      font-weight: normal;
+      line-height: 47px;
+      margin: 1rem;
+      
+    }
```

Comment:
probably some left padding for the whole component?

Reply:
sure, good idea! This component is created with the intention that the developer who wants to implement it in another component can make modifications to the design. as the width, padding, margin etc..

Reply:
Yea we should not and spacing inside the components if possible :)